src/Game.php

```diff
@@ -20,6 +22,36 @@ public function __construct(
         $this->eventManager = $eventManager;
     }
 
+    public static function getVersion(): string
+    {
+        // Conforms to semantic versioning.
+        return '0.0.1';
```

Comment:
Shouldn't we use "master" or "dev" as the version? It's not really a released version yet.